azure-functions/src/durable/history.rs

```diff
@@ -9,50 +9,48 @@ use serde_repr::Deserialize_repr;
 // i.e conversion { EventType = 0, EventId = 1, ...} => ExecutionStarted(ExecutionStartedEvent)
 // in future we can implement manual translation
 
-#[doc(hidden)]
 #[derive(Debug, Clone, Deserialize, PartialEq)]
 #[serde(rename_all = "PascalCase")]
-pub struct HistoryEvent {
-    pub(crate) event_type: EventType,
+pub(crate) struct HistoryEvent {
```

Comment:
You are exporting this module as `pub(crate)` [here](https://github.com/peterhuene/azure-functions-rs/pull/394/commits/773c739aeb770ae2f2ad7857e9af7b6ad20135b6#diff-548c8a0d6b23621afce6b271f710d622R26) so it will be accessible only inside this crate. Should you make it just `pub` here? The same comment for every time in history mod

Reply:
I believe the compiler needs to find at least one type within the module of the same visibility when reexporting using a `use`.  I'll play around with it, though.

Reply:
You're right.  I'll fix it so that they're `pub`.